subsys/logging/Kconfig.formatting

```diff
@@ -109,6 +109,14 @@ config LOG_DICTIONARY_SUPPORT
 
 	  This should be selected by the backend automatically.
 
+config LOG_CUSTOM_SUPPORT
+	bool "Custom support"
```

Comment:
too generic name. Something like `LOG_CUSTOM_FORMAT_SUPPORT` and `Custom formatter support` would be better.